src/huntsman/pocs/filterwheel/zwo.py

```diff
@@ -0,0 +1,32 @@
+from panoptes.pocs.filterwheel.zwo import FilterWheel as ZWOFilterWheel
+
+
+class FilterWheel(ZWOFilterWheel):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _move_to(self, position):
+
+        # Implement workaround for EFW library problem where cannot move to next-lowest position
+        # The workaround is to move to a different position before going to that position
+        # TODO: Remove this code when EFW library gets fixed!
+        n_filters = len(self._filter_names)
+        new_pos_int = self._parse_position(position)
+        diff = new_pos_int - self.position
+        if diff in (1, 1-n_filters):  # Problem only when this is True
```

Comment:
Pep8: `1 - n_filters`

Reply:
What are the possible filter positions? This is check for `1` and some other negative number (`-7`), so I'm assuming the first and or last position. But is that zero based? And should it be the absolute value?

Reply:
The problem is that the filterwheel can't rotate a full turn minus one position. i.e. it can't go from position 3 to 2 or from 5 to 4 without raising an error, which is in the EFW library (we told them about it but no response). The problem also exists if we go from 1 to 7 (i.e. the number of filters), in which case the difference is `1-n_filters`.